src/components/binding/binding.js

```diff
@@ -90,9 +92,52 @@ genotet.BindingView = function(viewName, params) {
   /** @protected {genotet.BindingRenderer} */
   this.renderer = new genotet.BindingRenderer(this.container, this.data);
 
+  /**
```

Comment:
This is not good encapsulation. link manager need to talk to views rather than the view's attributes.

linkManager has its own link handling inside it, which views don't care about.
views have their own attributes (e.g. loader) inside them, which linkManager doesn't care about.

LinkManager sends signal to let view do something.
LinkManager listens for signals fired on views to know what happened on views.